### PR TITLE
Add an injected service to the sample.

### PR DESCRIPTION
**Pull request by runningcode**

### CODE REVIEW

File: integration-tests/android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -14,5 +14,6 @@
         <category android:name="android.intent.category.DEFAULT"/>
       </intent-filter>
     </activity>
+      <service android:name=".ExampleService"/>
```
Comment (JakeWharton) [inline integration-tests/android/src/main/AndroidManifest.xml]: bad indent

File: integration-tests/android/src/main/java/com/example/ExampleService.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import android.content.Intent;
+import android.os.IBinder;
+import android.widget.Toast;
+import dagger.Module;
+import dagger.android.AndroidInjection;
+import dagger.android.ContributesAndroidInjector;
+
+import javax.inject.Inject;
+
+public class ExampleService extends Service {
```
Comment (JakeWharton) [inline integration-tests/android/src/main/java/com/example/ExampleService.java]: final

File: integration-tests/android/src/main/java/com/example/ExampleService.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    @Inject String string;
+
+    @Override
+    public IBinder onBind(Intent intent) {
```
Comment (JakeWharton) [inline integration-tests/android/src/main/java/com/example/ExampleService.java]: should be two-space indent

**Comment by runningcode** [comment]
updated style. :)

**Pull request merged**